### PR TITLE
Feat: Added Landing Page

### PR DESCRIPTION
**Pull request by GauravDe-git**
Made a responsive landing page for the component library.  Navbars change to menu bars after lowering the resolution.

page

image

image

### CODE REVIEW

**Comment by GauravDe-git** [comment]
Changed background color.
image

File: style.css [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+    width: 100%;
+    padding: 1.875rem 6.25rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+header .logo{
+    position: relative;
+    color: #000;
```
Comment (ruchiragoswami) [inline style.css]: Just drawing your attention, color is hard-coded here.

**Review by ruchiragoswami — APPROVED** [review]
Great Work Gaurav! I am approving with some general comments. 

1. Color is hard-coded at certain places, (on line numbers 45, 54, 86, 109, 121, 154). If this is temporary, it's fine. Otherwise, a better idea would be to make variables for colors and use them. 
2. Use of  'rem' units is preferred. So for anything more than 10px, most of us use 'rem'.

File: style.css (hunk 1 above)
Comment (GauravDe-git) [reply style.css]: Hey, thanks for approving the changes! For now, some of the colors are indeed temporary, and thanks for bringing up the usage of 'rem'. Will make the necessary changes.

**Review by shaik-saad — APPROVED** [review]

**Pull request merged**